lib/services/messenger/translators/github.ts

```diff
@@ -253,6 +253,7 @@ export class GithubTranslator extends TranslatorScaffold implements Translator {
 					hidden: _.isSet(metadata.hidden) ? metadata.hidden : false,
 					tags: thread.labels,
 					text: TranslatorScaffold.unixifyNewLines(text),
+					time: event.cookedEvent.data.created_at,
```

Comment:
Use `updated_at` rather than `created_at`. Any event that's updated will end up using the original creation date here, which may be far in the past.